tools/templates/nova_lfe/controller.dtl

```diff
@@ -0,0 +1,5 @@
+<html>
+<body>
+<h1>{{message}}</h1>
+</body>
+</html>
```

Comment:
Same with this file. We could use the one we already have to skip duplicates. But maybe we might want to say something specific for lfe here?

Reply:
We can use the one in nova directory.